package.json

```diff
@@ -24,14 +24,21 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
```

Comment:
It might be nice to have a "lint" script that runs eslint, and perhaps also a "format" script that runs prettier. But this recommendation isn't really necessary so feel free to skip it. Here's a link to an example (but again maybe not necessary): https://github.com/parcel-bundler/parcel/blob/v2/package.json.